scripts/search_start_points.py

```diff
@@ -0,0 +1,59 @@
+from scipy.optimize import minimize
+import numpy as np
+import os
+import csv
+
+from scripts.models import Qlearning, RescorlaWagner
+from scripts.player import Estimator, RealPlayer
+
+
+def search_optimal_parameters(estimator):
+    optimal_alpha = []
+    optimal_T = []
+    optimal_func_value = []
+    alpha_list = np.arange(0.0, 1.0, 0.01)
+    T_list = np.arange(1, 10, 0.1)
```

Comment:
those are not lists but numpy arrays also try to avoid naming variables with types